src/status_im/data_store/groups.cljs

```diff
@@ -0,0 +1,24 @@
+(ns status-im.data-store.groups
```

Comment:
IMO, `group` is too general name for the entity, we can add another entity that will require grouping in future and what then? I would prefer to define it as `contacts-group`.